host_core/lib/host_core/actors/actor_supervisor.ex

```diff
@@ -14,9 +14,7 @@ defmodule HostCore.Actors.ActorSupervisor do
     DynamicSupervisor.init(strategy: :one_for_one)
   end
 
-  @spec start_actor(bytes :: binary(), oci :: String.t(), count :: Integer.t()) ::
```

Comment:
Nit: don't we still want this spec here?

Reply:
Eventually. I couldn't get vscode to validate the spec. I'll try and check it back in and see if it works without warnings

Reply:
I had trouble getting dialyxir to recognize it but I think I got warnings that `Integer.t()` and `Map.t()` aren't real types and should be `integer()` and `map()`